PhoneNumberKit/UI/CountryCodePickerViewController.swift

```diff
@@ -0,0 +1,240 @@
+        searchController.searchBar.backgroundColor = .clear
+        navigationItem.searchController = searchController
+        definesPresentationContext = true
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    public override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if let nav = navigationController {
+            shouldRestoreNavigationBarToHidden = nav.isNavigationBarHidden
+            nav.setNavigationBarHidden(false, animated: true)
+        }
+        if let nav = navigationController, nav.isBeingPresented && nav.viewControllers.count == 1 {
+            navigationItem.setRightBarButton(cancelButton, animated: true)
+        }
+    }
+
+    public override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationController?.setNavigationBarHidden(shouldRestoreNavigationBarToHidden, animated: true)
+    }
+
+    @objc func dismissAnimated() {
+        dismiss(animated: true)
+    }
+
+    func country(for indexPath: IndexPath) -> Country {
+        isFiltering ? filteredCountries[indexPath.row] : countries[indexPath.section][indexPath.row]
+    }
+
+    public override func numberOfSections(in tableView: UITableView) -> Int {
+        isFiltering ? 1 : countries.count
+    }
+
+    public override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        isFiltering ? filteredCountries.count : countries[section].count
+    }
+
+    public override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: Cell.reuseIdentifier, for: indexPath)
+        let country = self.country(for: indexPath)
+
+        cell.textLabel?.text = country.prefix + " " + country.flag
+        cell.detailTextLabel?.text = country.name
+
+        cell.textLabel?.font = .preferredFont(forTextStyle: .callout)
+        cell.detailTextLabel?.font = .preferredFont(forTextStyle: .body)
+
+        return cell
+    }
+
+    public override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        if isFiltering {
+            return nil
+        } else if section == 0, hasCurrent {
+            return "Current"
+        } else if section == 0, !hasCurrent && hasCommon {
+            return "Common"
+        } else if section == 1 && hasCurrent && hasCommon {
+            return "Common"
+        }
+        return countries[section].first?.name.first.map(String.init)
+    }
+
+    public override func sectionIndexTitles(for tableView: UITableView) -> [String]? {
+        guard !isFiltering else {
+            return nil
+        }
+        var titles: [String] = []
+        if hasCurrent {
+            titles.append("•") // NOTE: SFSymbols are not supported otherwise we would use 􀋑
```

Comment:
Is it possible to use `AttributedString` here, just for iOS 13 so we can use SFSymbols

Reply:
There's no API to use attributed strings for index titles, typically you can put SFSymbols in text throughout the system so long as you are using the system font ... however that's just not the case for `sectionIndexTitles`